### ISSUE TITLE
[CFP] PragVue 2024

### ISSUE THREAD

**Issue by AloisSeckar** [vuejs/events#216]
## Conference 

We have started to discuss inside my company (Eviden Czech Republic), if we want/are able to host a conference about Vue.js here in Czechia. It looks like everyone wants to organize it, so we are starting some early preparations. I am not sure if here is the best place to start seeking for potential speakers from Vue.js community, but I hope at least someone would point me to the right direction 🥺 

**Where:** Prague, Czechia
**When:** mid september, 2024
**How long:** one day (around six hours)
**Type:** in-person (but online lectures also possible)

The bottom line of the meeting should be introducing Vue and the ecosystem to new people - to convince both developers and potentional customers they want their next web project to be created with Vue/Nuxt and co.

## CFP Link

TBA

## Speaker Amenities

TBA. But I expect if anyone wants to come, we **will** cover the travel, accomodation and meal expenses.

Also I can offer our beautiful city full of sightseeings for those, who haven't been there yet 🏙️

**Comment by kissu** [comment]
Hey 👋🏻  thanks for the PR!

> It looks like everyone wants to organize it, so we are starting some early preparations.
So I guess that the conference is official? The only thing that could happen is a cancelation in case you do not have enough speakers, right?

> I am not sure if here is the best place to start seeking for potential speakers from Vue.js community
Not really. But we could have a call sometime this week to talk about it in detail so that I could connect you with a lot of people interested. 😉 
Are you on Discord or Twitter? 😄 

> The bottom line of the meeting should be introducing Vue and the ecosystem to new people
This would be the topic of the conference mostly? 😄

**Comment by AloisSeckar** [comment]
Hi and thank you for quick reply and for the guidance.

> So I guess that the conference is official? The only thing that could happen is a cancelation in case you do not have enough speakers, right?

I understand it so. I hope I wasn't tasked to look for potential speakers just to tell them _"oh, sorry, my managers changed their mind"_ later 😨 

But I think once we have people, we will find a way.

>  Are you on Discord or Twitter? 😄

I have a Discord account `aloisseckar` although I don't use it much

> This would be the topic of the conference mostly? 😄

I meant it like we expect audience "outside of the bubble" that have barely heard of Vue so far rather than "Vue-nerds". So the talks should be more about _"Why should I choose Vue ecosystem over React and Angular"_ than delving deep into specific tooling or coding tricks. Although I can imagine some cool technologies would be that reason why, so it is not necessarily a contradiction.
